prescient/simulator/master_options.py

```diff
@@ -897,16 +383,12 @@ def _construct_inner_options_parser():
                                          dest='day_ahead_pricing',
                                          type='string',
                                          default='aCHP')
-    guiOverride['--day-ahead-pricing'] = {}
-    guiOverride['--day-ahead-pricing']['bpa'] = False
 
     solver_simulation_options.add_option('--relax-t0-ramping-initial-day',
                                          help="Relax t0 ramping for the initial RUC. Ignored for PH.",
                                          action='store_true',
                                          dest='relax_t0_ramping_initial_day',
```

Comment:
This option isn't used (yet). I assume you left it in because you plan to use it eventually? Might make sense to remove it until we use it, up to you.

Reply:
Removed.